gui/src/components/Backups.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
+        <h4 class="pt-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
```

Comment:
```suggestion
        <h4 class="pt-3">{{cluster.cluster_name}}</h4>
```

Very small thing but I think we need a little space under this title. Seems a bit packed.